docs/source/configuration/persisted-queries.mdx

```diff
@@ -66,6 +66,16 @@ persisted_queries:
 
 If used with the [`safelist`](#safelist) option, the router logs unregistered and rejected operations. With [`safelist.required_id`](#require_id) off, the only rejected operations are unregistered ones. If [`safelist.required_id`](#require_id) is turned on, operations can be rejected even when registered because they use operation IDs rather than operation strings.
 
+#### `local_manifest`
+
+Adding `local_manifest` to your `safelist` configuration allows you to use a local persisted query manifest in place of the hosted Uplink version. This is useful when you have an offline Enterprise license and are unable to use Uplink. This configuration option cause the router to not reload the manifest from the file system, so you will need to restart the router to apply changes.
```

Comment:
Needing to restart is fine for now, but I think it should support reloading in the future.
But I'd be more at ease if it went through the same mechanism as schema and config reload, because filesystem watching is full of fun edge cases

Reply:
Agree- good future improvement.